emission/storage/pipeline_queries.py

```diff
@@ -77,7 +77,7 @@ def mark_sectioning_done(user_id, last_trip_done):
         mark_stage_done(user_id, ps.PipelineStages.SECTION_SEGMENTATION, None)
     else:
         mark_stage_done(user_id, ps.PipelineStages.SECTION_SEGMENTATION,
-                        last_trip_done.data.end_ts + END_FUZZ_AVOID_LTE)
+                        last_trip_done['data']['end_ts'] + END_FUZZ_AVOID_LTE)
```

Comment:
Note to self: we need this because the `last_trip_done` is no longer an entry.

Note to @JGreenlee: we can also wrap the trip in a `Trip` object like you did with location, in case we need to keep it consistent with other stages.